Data_Collection.ipynb

```diff
@@ -0,0 +1,1759 @@
+       "28891    -4.581     1       0.0355        0.0132          0.000003    0.1390   \n",
+       "28892   -16.070     1       0.0561        0.9890          0.868000    0.1490   \n",
+       "28893   -23.025     0       0.0443        0.9640          0.696000    0.1030   \n",
+       "\n",
+       "       valence    tempo  duration_ms  time_signature  chorus_hit  sections  \\\n",
+       "0       0.6070  126.930       341333               3    35.14482        15   \n",
+       "1       0.1560   87.746       129267               4    43.11708         6   \n",
+       "2       0.1660   87.256       179507               3    33.79222         8   \n",
+       "3       0.1800   89.775       219453               3    20.08166         9   \n",
+       "4       0.5150  127.622       205867               4    64.98770         6   \n",
+       "...        ...      ...          ...             ...         ...       ...   \n",
+       "28889   0.0361   72.272       150857               4    24.30824         7   \n",
+       "28890   0.7400  119.985       152000               4    32.53856         8   \n",
+       "28891   0.6050  119.999       227760               4    20.73371         7   \n",
+       "28892   0.5600  120.030       213387               4    21.65301        14   \n",
+       "28893   0.0297   95.182       341396               4    71.05343        15   \n",
+       "\n",
+       "       target                                             bucket  \n",
+       "0           0  WednesdayNightPrayerMeeting-AlternativeTake-Ch...  \n",
+       "1           0       WhereTheLittleJesusSleeps-HarryBelafonte.wav  \n",
+       "2           0       JehovahTheLordWillProvide-HarryBelafonte.wav  \n",
+       "3           0                     SilentNight-HarryBelafonte.wav  \n",
+       "4           0                      TheBabyBoy-HarryBelafonte.wav  \n",
+       "...       ...                                                ...  \n",
+       "28889       0                             LotusFlowers-Yolta.wav  \n",
+       "28890       1                     CallingMySpirit-KodakBlack.wav  \n",
+       "28891       1                         TeenageDream-KatyPerry.wav  \n",
+       "28892       0                    StormyWeather-OscarPeterson.wav  \n",
+       "28893       0                                Dust-HansZimmer.wav  \n",
+       "\n",
+       "[28894 rows x 20 columns]"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "new_df = audio_data.copy()\n",
+    "new_df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "id": "1e46ad30-35e9-4f27-8ace-f7f0f12c21e9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_df = new_df.merge(genre_df, how=\"left\", on=[\"track\", \"artist\"]).merge(lyrics_df, how=\"left\", on=[\"track\", \"artist\"])"
```

Comment:
Why do you have to .merge twice?

Reply:
Oh I see, one is the genre df and one is lyrics

Reply:
The compiled dataset was organized perfectly so that we can perform feature extraction and preprocess the input!